docs/monitoring/configuration.rst

```diff
@@ -0,0 +1,206 @@
+.. |CONCH_available| image:: ../images/CONCH_available.jpg
+.. |CONCH_message| image:: ../images/CONCH_message.jpg
+.. |CONCH_slack| image:: ../images/CONCH_slack.jpg
+.. |north_audit| image:: ../images/north_audit.jpg
+.. |north_change_log| image:: ../images/north_change_log.jpg
+
+
+Configuration Changes
+=====================
+
+Whenever a configuration category is changed within Fledge an entry is written to the audit log that documents which category was changed along with the old and new value. For example if the configuration category for a south service called sine is changed a record as follows is written to the audit log with the source of the audit entry set to CONCH.
+
+.. code-block:: JSON
+
+   {
+      "category": "sine",
+      "items": {
+             "assetName": {
+                            "oldValue": "sinusoid",
+                            "newValue": "sinusoid3"
+                          }
+               }
+   }
+
+Since audit data can be used in the same way as an reading data within Fledge, this allows changes of configuration to be monitored and acted upon in the same way as any data that is changed within Fledge.
+
+Using Notifications
+-------------------
+
+As an example, let's assume we want to be informed if the configuration of one of the categories in Fledge has been altered. We can create a simple notification instance, using the Data Availability plugin that looks for any audit log entries with a source of CONCH.
+
++-------------------+
+| |CONCH_available| |
++-------------------+
+
+We then define the medium we want to use to deliver our notification, for ease of configuration we will use the Slack delivery plugin to do this.
+
++---------------+
+| |CONCH_slack| |
++---------------+
+
+Each time any user modifies a configuration item a slack message will be sent to alert that the configuration has been altered.
+
++-----------------+
+| |CONCH_message| |
++-----------------+
+
+This is a very simple message that only gives the information that a change has been made, however more sophisticated delivery mechanisms can be used that will detail the actual change.
+
+Sending To External Systems
+--------------------------
+
+Audit log data can also be sent to the north in the same way that reading data can. This can be used to send data to third party systems to maintain an change log of internal Fledge changes in other systems. In order to send audit log data to the north e merely setup a new north service or task and when we configure the north plugin we select the data source as *audit* rather than readings or statistics.
```

Comment:
- an change log -> a change log
- _... to the north e merely setup a new north ..._. Not clear what this should be.
- Start a new sentence at "When we configure the north plugin..."